docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -124,29 +124,105 @@ You can attach more than one action. Clicking the *Add action* button will promp
+
+Although available variables differ by rule type, when the action frequency is
+**For each alert**, all rule types pass the following variables:
+
 `alert.actionGroup`:: The ID of the action group of the alert that scheduled the action.
-`alert.actionSubgroup`:: The action subgroup of the alert that scheduled the action.
 `alert.actionGroupName`:: The name of the action group of the alert that scheduled the action.
+`alert.actionSubgroup`:: The action subgroup of the alert that scheduled the action.
 `alert.flapping`:: A flag on the alert that indicates whether the alert status is changing repeatedly.
+`alert.id`:: The ID of the alert that scheduled the action.
 
-`kibanaBaseUrl`:: The configured <<server-publicBaseUrl, `server.publicBaseUrl`>>. If not configured, this will be empty.
+[float]
+[role="child_attributes"]
+[[alert-summary-action-variables]]
+===== Action variables for summary of alerts
+
+NOTE: This type of action frequency is not available for all rule types.
+
+When the action frequency is **Summary of alerts**, rules pass the following
+variables:
+
+`alerts.all.count`:: The count of all alerts.
+
+`alerts.all.data`::
+An array of objects for all alerts. The following object properties are examples; it is not a comprehensive list.
++
+.Properties of the alerts.all.data objects
+[%collapsible%open]
+=====
+//# tag::alerts-data[]
+`kibana.alert.end`:: Datetime stamp of alert end.
+`kibana.alert.flapping`:: A flag on the alert that indicates whether the alert status is changing repeatedly.
+`kibana.alert.instance.id`:: ID of the source that generates the alert.
+`kibana.alert.reason`:: The reason of the alert (generated with the rule conditions).
+`kibana.alert.start`:: Datetime stamp of alert start.
+`kibana.alert.status`:: Alert status (for example, active or OK).
+//# end::alerts-data[]
+=====
+
+`alerts.new.count`:: The count of new alerts.
+
+`alerts.new.data`::
+An array of objects for new alerts. The following object properties are examples; it is not a comprehensive list.
++
+.Properties of the alerts.new.data objects
+[%collapsible]
+=====
+include::create-and-manage-rules.asciidoc[tag=alerts-data]
+=====
+
+`alerts.ongoing.count`:: The count of ongoing alerts.
+
+`alerts.ongoing.data`::
+An array of objects for ongoing alerts. The following object properties are examples; it is not a comprehensive list.
++
+.Properties of the alerts.ongoing.data objects
+[%collapsible]
+=====
+include::create-and-manage-rules.asciidoc[tag=alerts-data]
+=====
+
+`alerts.recovered.count`:: The count of recovered alerts.
+
+`alerts.recovered.data`::
+An array of objects for recovered alerts. The following object properties are examples; it is not a comprehensive list.
++
+.Properties of the alerts.recovered.data objects
+[%collapsible]
+=====
+include::create-and-manage-rules.asciidoc[tag=alerts-data]
+=====
 
 [role="screenshot"]
 image::images/rule-flyout-action-variables.png[Passing rule values to an action]
```

Comment:
Would this screenshot make more sense at the beginning of the "Action variables" section?

Reply:
Agreed, done in 9063cca114a1fb8d71a47a113a6c92623e8f0f50